modules/quickstart/pages/dfx-deploy-quickstart.adoc

```diff
@@ -0,0 +1,389 @@
++
+[source,bash]
+----
+dfx start --background
+----
+. Register, build, and deploy all of the canisters specified in the default `+dfx.json+` file by running the following command:
++
+[source,bash]
+----
+dfx deploy my_counter_hello
+----
++
+In this case, we specify a canister name to register, build, and deploy because we haven't added any front-end code to the project.
+
+==== Test the application in a terminal
+
+To test the application in a terminal:
+
+. Call the `+whoami+` function with your name as an argument by running a command similar to the following:
++
+[source,bash]
+----
+dfx canister call my_counter_hello whoami '("YOUR-NAME")'
+----
++
+The command displays output similar to the following:
++
+[source,bash]
+----
+("Welcome to the simplified developer preview, Jane: 1")
+----
++
+Repeat the call the `+whoami+` function a few times to see the counter value increment.
+. Gall the `+get+` function by running the following command:
++
+[source,bash]
+----
+dfx canister call my_counter_hello get
+----
++
+The command displays output similar to the following:
++
+[source,bash]
+----
+("Welcome to the simplified developer preview, your counter value is: 3")
+----
+
+==== Test functions in a browser
+
+To test the application in a browser:
+
+. Copy the canister identifier for the `+my_counter_hello+` canister.
+. Open a browser and navigate to the default host name and port.
+. Add the `+candid+` endpoint followed by `canisterId`+ and the canister identifier for the `+my_counter_hello+` canister.
++
+If you are using the default local host and port, you should have a URL similar to the following:
++
+....
+http://127.0.0.1:8000/candid?canisterId=75hes-oqbaa-aaaaa-aaaaa-aaaaa-aaaaa-aaaaa-q
+....
+. Type a name for the `whoami` function, then click *Call* to see the result. 
+. Delete the text from the `whoami` function, then click *Lucky* to generate a random text value and see the result.
+. Click *Query* for the `get` function.
+
+==== Stop the local network
+
+To stop the local network:
+
+. In the terminal that displays network operations, press Control-C to interrupt the local network process.
+. Stop the local {IC} network by running the following command:
++
+[source,bash]
+----
+dfx stop
+----
+
+== Command-line reference
+
+The {sdk-short-name} introduces the following new `+dfx+` subcommand:
+
+[width="100%",cols="<32%,<68%",options="header"]
+|===
+|command |Description
+
+|`+deploy+` |Deploys all or a specific canister from the code in your project. 
+By default, all canisters are deployed.
+
+|===
+
+=== Basic usage
+
+[source,bash]
+----
+dfx deploy [flag] [option] [--all | canister_name]
+----
+
+=== Flags
+
+You can use the following optional flags with the `+dfx deploy+` command.
+
+[width="100%",cols="<32%,<68%",options="header"]
+|===
+|Flag |Description
+|`+-h+`, `+--help+` |Displays usage information.
+
+|`+-V+`, `+--version+` |Displays version information.
```

Comment:
The `-V` / `--version` option applies to `dfx`, so it has to come before `deploy`.  I don't know why `dfx deploy -h` lists it as a `FLAG` for this command.